AppCenterData/AppCenterDataTests/MSDBDocumentStoreTests.m

```diff
@@ -201,8 +201,10 @@ - (void)testReadUserDocumentFromLocalDatabaseNotFound {
 - (void)testUpsertReplacesCorrectlyInAppStorage {
 
   // If
-  MSDocumentWrapper *expectedDocumentWrapper = [MSDocumentUtils documentWrapperFromData:[self jsonFixture:@"validTestDocument"]
+  MSDocumentWrapper *expectedDocumentWrapper = [MSDocumentUtils documentWrapperFromData:[self jsonFixture:@"validTestAppDocument"]
                                                                            documentType:[MSDictionaryDocument class]
+                                                                              partition:@"readonly"
```

Comment:
You should use the constants for the partition name.

Reply:
Done.